dotCMS/src/main/java/com/dotcms/mock/request/MockParameterRequest.java

```diff
@@ -24,6 +27,13 @@ public MockParameterRequest(HttpServletRequest request) {
     public MockParameterRequest(HttpServletRequest request, Map<String, String> setMe) {
         super(request);
         HashMap<String, String> mutable = new HashMap<String, String>();
+        
+        List<NameValuePair> additional = URLEncodedUtils.parse(request.getQueryString(), Charset.forName("UTF-8"));
+        for(NameValuePair nvp : additional) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'nvp' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5391667)